MVVM-C/Controllers/Article Details/DetailViewVC.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  DetailView.swift
+//  GoTApp
+//
+//  Created by Jakub Homik on 07/04/2020.
+//  Copyright © 2020 Jakub Homik. All rights reserved.
+//
+
+import UIKit
+
+final class DetailViewVC: UIViewController {
+    
+    private let articleLabel = GoTLabel(textAlignment: .center, fontSize: 24)
+    private let imageArticle = GoTImageThumbnail(frame: .zero)
+    private let abstractLabel = GoTLabel(textAlignment: .center, fontSize: 16)
+    private let safariButton = UIButton()
+    var viewModel: ArticleViewModel?
```

Comment:
why you set this property as an optional ? If you initializing it I think you dont need it? Or its just because we dont know if there will be any data? But in this situation why we even initializing it ?

Reply:
I have set this property optional because it's being inject via property injection, but we could change it to be injected via init injection, but even in that case we would have to make it optional, because we won't be able to override the required initializers from the `UIViewController` class, so in that case this value could be optional if someone dont uses the correct init method.
In the currently scenario I've made it optional to avoid a crash if someone forgets to set its value.